deployments/vanilla/terraform/main.tf

```diff
@@ -112,6 +112,18 @@ module "eks_blueprints_outputs" {
   tags = local.tags
 }
 
+module "nvidia_device_plugin" {
+  count = var.install_nvidia_device_plugin ? 1 : 0
```

Comment:
install this automatically if var.node_instance_type starts with `p` or `g`?

Reply:
done